deploy/packaging/talaria_spruce.yaml

```diff
@@ -305,6 +302,10 @@ inbound:
   # WARNING: This is an example auth token. DO NOT use this in production.
   authKey: (( grab $AUTH_HEADER || "dXNlcjpwYXNz" ))
 
+  # requestTimeout is the timeout for all inbound HTTP requests.
+  # (Optional) defaults to 30s
```

Comment:
Hi @joe94,
The default timeout is 120s. we need to update properties or need to change the value [here](https://github.com/xmidt-org/talaria/blob/d86b9d63618dc32e81394781379964b3b8f0522e/primaryHandler.go#L64).

Reply:
Yes, thank you!